.github/workflows/merge-gate.yml

```diff
@@ -33,13 +33,13 @@ jobs:
     permissions:
       contents: read
 
-  saas-tests:
-    name: SaaS
-    needs: [ run-slow-tests ]
-    uses: ./.github/workflows/tests-saas.yml
-    secrets: inherit
-    permissions:
-      contents: read
+#  saas-tests:
```

Comment:
Would it be useful to put a comment here? Like when this could be used again or that breaking so often not worth running atm.